onnx/test/test_backend_onnxruntime.py

```diff
@@ -298,6 +298,23 @@ def run_node(cls, node, inputs, device=None, outputs_info=None, **kwargs):
             "|qlinearmatmul_3D_uint8_float32"
             ")"
         )
+    if ort_version is not None and ort_version < Version("1.18"):
+        backend_test.exclude(
```

Comment:
Some of these tests are disabled for 1.17. Maybe the list for 1.17 can reduced to those failing only for 1.17.

Reply:
Is it better to keep the 1.17 list as it is so that when ort 1.16 is installed one can run the test without failure?

Reply:
> Some of these tests are disabled for 1.17. Maybe the list for 1.17 can reduced to those failing only for 1.17.

I think this is the list for 1.17. Since: ort_version < Version("1.18")